composer.json

```diff
@@ -88,7 +88,6 @@
         "drupal/migrate_tools": "8.2.0-beta1",
         "drupal/oauth": "8.1.x-dev",
         "drupal/page_manager": "8.1.0-alpha23",
-        "drupal/panels": "8.3.x-dev",
```

Comment:
Yeah - and it was in conflict with the tag that Lightning uses.

I think it's best practice to declare your dependencies as well though to avoid surprises. So this should probably be:

    - "drupal/panels": "8.3.x-dev",
    + "drupal/panels": "8.3.0-beta4",

If there's something in head that DF needs, we can patch it in the extras section.